plugins/inputs/logparser/logparser_test.go

```diff
@@ -218,11 +226,14 @@ func TestGrokParseLogFiles_TimestampInEpochMilli(t *testing.T) {
 		},
 		map[string]string{
 			"response_code": "200",
-			"path":          thisdir + "testdata/test_c.log",
+			"path":          thisdir + "testdata" + pathSeparatorString + "test_c.log",
 		})
 }
 
 func getCurrentDir() string {
 	_, filename, _, _ := runtime.Caller(1)
+	if runtime.GOOS == "windows" {
+		filename = filepath.FromSlash(filename)
+	}
```

Comment:
No need to check for the OS as `filepath.FromSlash()` handles this internally.

Reply:
Right! Function changed.